src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -1104,6 +1103,117 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
+            it.destination.portNumber == updatedFlowDstEndpoint.destination.portNumber
+            it.destination.vlanId == updatedFlowDstEndpoint.destination.vlanId
+        }
+
+        and: "Flow path is not rebuild"
+        Wrappers.timedLoop(rerouteDelay) {
+            pathHelper.convert(northbound.getFlowPath(flow.flowId)) == currentPath
+        }
+
+        then: "Update the flow: port number and vlanId on the src/dst endpoints"
+        def updatedFlow = flow.jacksonCopy().tap {
+            it.source.portNumber = 10 // traffgen port
+            it.source.vlanId = updatedFlowDstEndpoint.source.vlanId + 1
+            it.destination.portNumber = 10 // traffgen port
+            it.destination.vlanId = updatedFlowDstEndpoint.destination.vlanId + 1
+        }
+        flowHelperV2.updateFlow(flow.flowId, updatedFlow)
+
+        then: "Flow is really updated"
+        with(northboundV2.getFlow(flow.flowId)) {
+            it.source.portNumber == updatedFlow.source.portNumber
+            it.source.vlanId == updatedFlow.source.vlanId
+            it.destination.portNumber == updatedFlow.destination.portNumber
+            it.destination.vlanId == updatedFlow.destination.vlanId
+        }
+
+        and: "Flow path is not rebuild"
+        Wrappers.timedLoop(rerouteDelay + WAIT_OFFSET / 2) {
+            pathHelper.convert(northbound.getFlowPath(flow.flowId)) == currentPath
+        }
+
+        and: "Flow is valid"
+        northbound.validateFlow(flow.flowId).each { direction -> assert direction.asExpected }
+
+        and: "System allows traffic on the flow"
+        def traffExam = traffExamProvider.get()
+        def examFlow = new FlowTrafficExamBuilder(topology, traffExam).buildBidirectionalExam(
+                flowHelperV2.toV1(updatedFlow), 100, 5
+        )
+        withPool {
+            [examFlow.forward, examFlow.reverse].eachParallel { direction ->
+                def resources = traffExam.startExam(direction)
+                direction.setResources(resources)
+                assert traffExam.waitExam(direction).hasTraffic()
+            }
+        }
+
+        and: "All involved switches pass switch validation"
+        withPool {
+            currentPath*.switchId.eachParallel { SwitchId swId ->
+                with(northbound.validateSwitch(swId)) { validation ->
+                    validation.verifyRuleSectionsAreEmpty(["missing", "excess", "misconfigured"])
+                    validation.verifyMeterSectionsAreEmpty(["missing", "excess", "misconfigured"])
+                }
+            }
+        }
+        def involvedSwitchesPassSwValidation = true
+
+        cleanup: "Revert system to original state"
+        flow && flowHelperV2.deleteFlow(flow.flowId)
+        northbound.deleteLinkProps(northbound.getAllLinkProps())
+        !involvedSwitchesPassSwValidation && currentPath*.switchId.each { SwitchId swId ->
+            northbound.synchronizeSwitch(swId, true)
+        }
+        database.resetCosts()
```

Comment:
not required

Reply:
you are right, it is needless
 allLinkProps are already deleted
fixed